src/service.js

```diff
@@ -0,0 +1,49 @@
+import { loadCoursesAction } from './store/courses/actions';
+import { loadAuthorsAction } from './store/authors/actions';
+import { loginUserAction } from './store/user/actions';
+
+/*export const getCourses = () => async (dispatch) => {
+	try {
+		await fetch('http://localhost:4000/courses/all')
+			.then((Response) => Response.json())
+			.then((data) => dispatch({ type: GET_COURSES, payload: data }));
+	} catch (error) {}
+};*/
+
+export const loadCourses = (dispatch) => {
+	try {
+		fetch('http://localhost:4000/courses/all')
+			.then((Response) => Response.json())
+			.then((data) => dispatch(loadCoursesAction(data.result)));
+	} catch (error) {}
```

Comment:
this was not part of the task, but usually it's a good idea to at least `console.error` inside these catch blocks for easier troubleshooting